README.md

```diff
@@ -32,6 +32,17 @@ The connector supports both password and private key authentication methods.
 
 ## Destination
 
+Destination connects to a remote server. It takes an `opencdc.Record`, extracts filename from the metadata and upload the file to the remote server. The connector supports both password and private key authentication methods.
+
```

Comment:
Chunking is a nice feature and it would be good to mention in the readme that we support it and in which way (what metadata is needed, and so on), so that other sources (not only SFTP) can use it.

Reply:
Mentioned.